Wflow/src/sbm.jl

```diff
@@ -32,6 +33,12 @@ function LandHydrologySBM(dataset::NCDataset, config::Config, domain::DomainLand
     else
         snow = NoSnowModel()
     end
+    do_lst = get(config.model, "lst__flag", false)::Bool
```

Comment:
Suggest to use a more explicit name for this model setting:
```suggestion
    do_lst = get(config.model, "land_surface_temperature__flag", false)::Bool
```

Reply:
adopted, thanks